site/content/en/docs/adopters/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Adopters"
+linkTitle: "Adopters"
+weight: 10
+description: >
+  Where and how Kueue is used
+---
+
+Below are the adopters of Kueue, and we show where and how Kueue is used. 
+If you are using Kueue, please add yourself to the following list.
+
+## Adopters
+
+|                     Organization                     |             Description             |         Using Integration          |                 Contact                  |
+|:----------------------------------------------------:|:-----------------------------------:|:----------------------------------:|:----------------------------------------:|
+| [CyberAgent, Inc.](https://www.cyberagent.co.jp/en/) | On-premise ML Platform for internal | batch/job <br> kubeflow.org/mpijob | @[tenzen-y](https://github.com/tenzen-y) |
```

Comment:
```suggestion
| [CyberAgent, Inc.](https://www.cyberagent.co.jp/en/) | On-premise ML Platform | batch/job <br> kubeflow.org/mpijob | @[tenzen-y](https://github.com/tenzen-y) |
```

Reply:
@alculquicondor I gave feedback from my company's team that we'd like to emphasize that our ML Platform is only for internal use.

Do you have any nice idea instead `internal`?

Reply:
Or, how about `On-premise ML Platform for internal use`?

Reply:
I guess what you are trying to say is that you don't offer the platform as a service for other companies.

Maybe we can have another column for `Type`, next to organization, which can be `End User` vs `Provider`. You fall within `End User`.

Otherwise, what you propose is fine.

Reply:
> I guess what you are trying to say is that you don't offer the platform as a service for other companies.

Yes, that's right.

> Maybe we can have another column for Type, next to organization, which can be End User vs Provider. You fall within End User.

It sounds great!

Reply:
done.